### DISCUSSION TITLE
源编辑光标问题

### DISCUSSION THREAD

**Discussion by ckstc** [xiaohucode/yidaRule#81] [Ideas]
不能直接选择光标位置，点击距离与当前光标过近时，会弹出选项栏，而非定位到点选位置。
![image](https://github.com/xiaohucode/yidaRule/assets/33508773/1fce955c-1f40-43aa-8807-c2d0b05f5121)

选择特定内容后，不能直接拖动光标，必须先手指按到光标上过一段时间才能拖动
![image](https://github.com/xiaohucode/yidaRule/assets/33508773/90c55880-260d-459f-981b-991fe2292a76)

**Comment by 13316282905** [comment]
番茄小说规则软件就像这些光标一样，看不了正文
